mp/bundle

```diff
@@ -13,7 +13,7 @@ AUTH_HOST=${AUTH_HOST//\//\\/}
 RELEASE_VERSION=${RELEASE_VERSION//\//\\/}
 TRACKING_PAGE_HOST=${TRACKING_PAGE_HOST//\//\\/}
 
-docker compose run --rm bundler bash -c "
+docker compose run --rm -T bundler bash -c "
```

Comment:
I made this changes because of this new `mp watch` command I added. Basically, `mp watch` will watch will bundle the schema upon file changes in the _specification_ directory. Without the _-T_ some weird docker warnings were shown.